frontend/src/features/leases/list/LeaseFilter/LeaseFilter.tsx

```diff
@@ -66,29 +66,57 @@ const idFilterPlaceholders = {
  */
 export const LeaseFilter: React.FunctionComponent<ILeaseFilterProps> = ({ filter, setFilter }) => {
   const onSearchSubmit = (values: ILeaseFilter, { setSubmitting }: any) => {
-    let selectedPrograms: MultiSelectOption[] = multiselectRef.current?.getSelectedItems();
-    let programIds = selectedPrograms.map<string>(x => x.id);
-    values = { ...values, programs: programIds };
+    const selectedPrograms: MultiSelectOption[] = multiselectProgramRef.current?.getSelectedItems();
+    const selectedStatuses: MultiSelectOption[] = multiselectStatusRef.current?.getSelectedItems();
+    const programIds = selectedPrograms.map<string>(x => x.id);
+    const statuses = selectedStatuses.map<string>(x => x.id);
+    values = { ...values, programs: programIds, leaseStatusTypes: statuses };
     setFilter(values);
     setSubmitting(false);
   };
   const resetFilter = () => {
-    multiselectRef.current?.resetSelectedValues();
+    multiselectProgramRef.current?.resetSelectedValues();
+    setInitialSelectedStatus(
+      statusFilterOptions.filter(x => defaultFilter.leaseStatusTypes.includes(x.id)),
+    );
+
     setFilter(defaultFilter);
   };
 
-  const multiselectRef = React.createRef<Multiselect>();
+  const multiselectProgramRef = React.createRef<Multiselect>();
+  const multiselectStatusRef = React.createRef<Multiselect>();
 
   const lookupCodes = useLookupCodeHelpers();
 
-  const leaseStatusOptions = lookupCodes.getByType(LEASE_STATUS_TYPES).map(c => mapLookupCode(c));
   const regionOptions = lookupCodes.getByType(REGION_TYPES).map(c => mapLookupCode(c));
 
+  const leaseStatusOptions = lookupCodes.getByType(LEASE_STATUS_TYPES);
+
   const leaseProgramTypes = lookupCodes.getByType(LEASE_PROGRAM_TYPES);
   const programFilterOptions: MultiSelectOption[] = leaseProgramTypes.map<MultiSelectOption>(x => {
     return { id: x.id as string, text: x.name };
   });
 
+  const statusFilterOptions: MultiSelectOption[] = leaseStatusOptions.map<MultiSelectOption>(x => {
+    return { id: x.id as string, text: x.name };
+  });
+  const initialLeaseStatusList = statusFilterOptions.filter(x =>
+    defaultFilter.leaseStatusTypes.includes(x.id),
+  );
+
+  const [selectedStatus, setInitialSelectedStatus] = useState<MultiSelectOption[]>(
+    initialLeaseStatusList,
+  );
+
+  // Necessary since the lookup codes does not work properly with the multi-select on first render
+  useEffect(() => {
+    setInitialSelectedStatus([{ id: 'ACTIVE', text: 'Active' }]);
+  }, []);
```

Comment:
likely the issue is that the lookup codes haven't been loaded yet, might make more sense to have this useeffect set the initial status until the lookup codes have loaded, or block somewhere above this to ensure lookup codes have loaded before executing this logic.

Reply:
Or this, the above comment makes it seem like the reason why it doesn't work is unknown.